src/controller/cve.controller/cve.controller.js

```diff
@@ -271,14 +271,15 @@ async function submitCna (req, res, next) {
     const owningCna = await orgRepo.findOneByUUID(cveId.owning_cna)
     const assignerShortName = owningCna.short_name
     const cnaContainer = req.ctx.body.cnaContainer
+    const dataUpdated = (new Date()).toISOString()
     const additionalCveMetadataFields = {
       assignerShortName: assignerShortName,
       requesterUserId: cveId.requested_by.user,
-      dateReserved: (cveId.reserved).toISOString(),
-      datePublished: (new Date()).toISOString()
+      dateReserved: dataUpdated,
```

Comment:
dateReserved should come from the date cve-id.reserved, not be set to the current timestamp